test/models/edition_test.rb

```diff
@@ -222,6 +222,30 @@ def draft_second_edition_from(published_edition)
     assert_equal new_edition.whole_body, edition.whole_body
   end
 
+  test "Cloning from LicenceEdition into AnswerEdition" do
+    edition = FactoryGirl.create(
+      :licence_edition,
+      state: "published",
+      panopticon_id: @artefact.id,
+      version_number: 1,
+      department: "Test dept",
+      licence_overview: "I am a test overview",
+      licence_identifier: "Test identifier",
+      licence_short_description: "I am a test short description",
+      will_continue_on: "test will continue on",
+      continuation_link: "https://github.com/alphagov/"
+    )
+    new_edition = edition.build_clone AnswerEdition
+
+    assert_equal new_edition.class, AnswerEdition
+    assert_equal new_edition.version_number, 2
```

Comment:
Thanks! Amended now, will do a separate PR to amend the order in the other neighbouring tests.